hack/testing-olm/utils

```diff
@@ -47,159 +47,9 @@ get_all_logging_pod_logs() {
   set -e
 }
 
-
-deploy_marketplace_operator(){
```

Comment:
comparable logic is already covered in "catalog_deploy.sh" correct?

Reply:
yeah this wasn't even being called from what i saw.